### PR TITLE
refactor: Update code according to IntelliJ analyze suggestions

### PR DESCRIPTION
**Pull request by magicus**
Time for somewhat belated spring cleaning!

### CODE REVIEW

File: common/src/main/java/com/wynntils/core/features/overlays/ContainerOverlay.java [hunk 1]
```diff
@@ -172,7 +172,7 @@ public void updateSize(Overlay overlay, OverlaySize containerSize, OverlaySize i
             }
         }
 
-        public OverlayPosition getChildPosition(
+        protected OverlayPosition getChildPosition(
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/core/features/overlays/ContainerOverlay.java]: Is Qodana right?

File: common/src/main/java/com/wynntils/features/map/MinimapFeature.java [hunk 2]
```diff
@@ -560,19 +560,19 @@ public enum MapBorderType {
             this.groovesSize = groovesSize;
         }
 
-        public Texture texture() {
+        protected Texture texture() {
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/features/map/MinimapFeature.java]: Same here? Should these be private?

File: common/src/main/java/com/wynntils/screens/maps/widgets/PoiManagerWidget.java [hunk 3]
```diff
@@ -30,15 +30,15 @@
...
-    private Button upButton;
-    private Button downButton;
-    private int row;
+    private final CustomPoi poi;
+    private final Button editButton;
+    private final Button deleteButton;
+    private final Button upButton;
+    private final Button downButton;
+    private final int row;
     private CustomColor color;
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/screens/maps/widgets/PoiManagerWidget.java]: Quite unfortunate ordering, can you move color to be last? (What is even setting color...)

**Review by kristofbolyai — APPROVED** [review]
Looks good overall!

File: common/src/main/java/com/wynntils/utils/type/TimedSet.java [hunk 4]
```diff
@@ -99,9 +99,9 @@ public void remove() {
     }
 
     public final class TimedEntry {
-        final T entry;
-        final long creation;
-        final long duration;
+        private final T entry;
+        private final long creation;
+        private final long duration;
```
Comment (Mcrtin) [inline common/src/main/java/com/wynntils/utils/type/TimedSet.java:104]: i don't know if its worth the time but can't this TimedEntry be a record?

**Review by kristofbolyai — APPROVED** [review]

**Comment by magicus** [comment]
Thanks for finishing up this for me! I had it on my list to get back to, but I never got that far...

**Pull request merged**